integ-test/src/test/java/org/opensearch/sql/legacy/SourceFieldIT.java

```diff
@@ -15,9 +15,9 @@
 import org.junit.Test;
 import org.opensearch.action.search.SearchResponse;
 import org.opensearch.common.xcontent.LoggingDeprecationHandler;
-import org.opensearch.common.xcontent.NamedXContentRegistry;
 import org.opensearch.common.xcontent.XContentFactory;
-import org.opensearch.common.xcontent.XContentParser;
+import org.opensearch.core.xcontent.NamedXContentRegistry;
```

Comment:
reorder?

Reply:
+1 on the reorder, is there a linter that can be run?

Reply:
Oh, right, checkstyle errors are ignored in IT.